changelog.txt

```diff
@@ -1,3 +1,4 @@
+* Ignore duplicate certificate authentication challenge in system webview.
```

Comment:
Nit - did you mean embedded webview, as the changes you made are within embeddedWebview domain?

Reply:
Hey Peter. The cert challenge is received in embedded webview but is actually handled in system webview which is where we need to prevent duplicate handling.